plugins.txt

```diff
@@ -63,7 +63,7 @@ metrics:4.0.2.7
 opentelemetry:0.13
 pipeline-build-step:2.13
 pipeline-github:2.7
-pipeline-graph-analysis:1.10
+pipeline-graph-analysis:1.11
```

Comment:
Not sure how to read the changelog without any deep knowledge of the plugin: let's roll (https://github.com/jenkinsci/pipeline-graph-analysis-plugin/blob/master/CHANGELOG.md#111-02-jun-2021)